types/index.d.ts

```diff
@@ -276,7 +270,7 @@ declare namespace unified {
    *
    * @typeParam P Processor settings
    */
-  type PluggableList<P = Settings> = Array<Pluggable<[any?], P>>
+  type PluggableList<P = Settings> = Array<Pluggable<any[], P>>
```

Comment:
now that touples are checked this needed to be updated to ensure this test:
```ts
processor.use([
  [pluginWithTwoSettings, processor, typedSetting],
  [pluginWithTwoSettings, processor, typedSetting]
])
```
a list of tuples with multiple settings, continues working